rebuild/src/addon.cc

```diff
@@ -0,0 +1,85 @@
+#include "addon.h"
+
+bool is_zero_bytes(const uint8_t *data,
+                   const size_t start_byte,
```

Comment:
can we avoid using tabs and sort of `compress` the spacing in formatting

Reply:
The lint is driven by the vscode c++ plugin defaults.  I will research how to update that.  In `c-kzg` there was a `c` lint library they used so I look at bringing in the same one.

Reply:
I added .clang-format.  We can change lint rules as desired now.  I set using the google standard and added a couple of rules I found in the c-kzg.  Feel free to make suggestions if you would like anything updated for readability.  Here are the rules:

https://clang.llvm.org/docs/ClangFormatStyleOptions.html